turbo.json

```diff
@@ -2,8 +2,16 @@
   "$schema": "https://turbo.build/schema.json",
   "pipeline": {
     "build": {
-      "outputs": ["out/**", "dist/**"],
-      "dependsOn": ["^build"]
+      "outputs": ["dist/**"],
```

Comment:
Do we need to clean a Turbo local cache for this to work?

It's still telling me cache hit for antlr4-c3 but the dist folder will not show up.

I'd say maybe this would need to be `"outputs": ["packages/*/dist/**", "vendor/*/dist/**"]` instead

Reply:
Could you try`rm -rf node_modules` and then run the build scripts without using `--force`?